packages/terra-site/src/examples/card/CardPaddingHR.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Card from 'terra-card';
+
+const CardPaddingHR = () => (
+  <div>
+    <Card>
+      <Card.Body>Hello World Above The Line!!</Card.Body>
+      <hr />
```

Comment:
Lets add custom styling on the `<hr>` for this example that makes the divider look like it fits better with the default card styling: 

```
border: 0 none;
border-top: 1px solid #c8cacb;
box-sizing: border-box;
height: 1px;
margin: 0;
```